dependencies.yaml

```diff
@@ -74,6 +74,7 @@ devel_no_selenium_requires:
   curl:
   rsync:
   postgresql-devel:
+  openssh-common:
```

Comment:
oops, sorry. I think this should go to "test_requires" instead so that also the tests during OBS build time can work

Reply:
Addressed in c590841